src/main/java/org/fundacionjala/at13/spaceinvaders/GameOver.java

```diff
@@ -26,9 +27,11 @@ public GameOver() {
     }
 
     /**
-     * Shows game over interface.
+    * Shows game over interface.
```

Comment:
Add a space in beginning for alienate the comment

Reply:
Thanks. I just did it.